lib/reline/line_editor.rb

```diff
@@ -250,7 +250,8 @@ def reset_variables(prompt = '', encoding:)
     @resized = false
     @cache = {}
     @rendered_screen = RenderedScreen.new(base_y: 0, lines: [], cursor_y: 0)
-    @past_lines = []
+    @past_lines = [[[""], 0, 0]]
+    @position = 0
```

Comment:
The instance variable name `position` feels too broad. Since the `line_editor.rb` is a large file, we want to narrow down what `position` refers to. For example, if it’s related to `input_lines`, we could use `input_lines_position`.

Reply:
I went with input_lines_position as well.

[Rename @position to @input_lines_position](https://github.com/ruby/reline/pull/707/commits/7aee2caa05b0197607557eef80c6816b30b2ef44)